apps/meteor/app/authentication/server/startup/index.js

```diff
@@ -40,7 +40,7 @@ Accounts.config({
  *
  * we are removing the status here because meteor send 'offline'
  */
-Object.assign(Accounts._defaultPublishFields.projection, (({ status, ...rest }) => rest)(getBaseUserFields()));
+Object.assign(Accounts._defaultPublishFields.projection, (({ status, ...rest }) => rest)(getDefaultUserFields()));
```

Comment:
I noticed that `getDefaultUserFields` also includes `'services.password.bcrypt'`. I don't think this field is supposed to be passed to client, did you ensure this doesn't happen?

Reply:
Good catch @gabriellsh !

@yash-rajpal , assuming your intentions here were to include the 2fa info. I would suggest including/extending the attire manually instead of projecting all the other attrs present in the `getDefaultUserFields()` func.

What do you guys think?

Reply:
Ok so I confirmed and this shouldn't really be happening. What I suggest:

Extend `getBaseUserFields` to include `'services.totp.enabled': 1, 'services.email2fa.enabled': 1,` and remove those from `getDefaultUserFields`.

This should suffice for now.

Reply:
As discussed on DM, pushed the fix.